### ISSUE TITLE
Update Section 3.3. purpose to refer to WCAG autocomplete values 

### ISSUE THREAD

**Issue by becka11y** [w3c/personalization-semantics#139]
Right now we reference the HTML5.2 autofill values in section 3.3.3.  

It has come up at one of our meetings that we should be referencing the autofill values that WCAG 2.1 has adopted.  We need to update the spec to state this. 

https://w3c.github.io/personalization-semantics/content/index.html#purpose-explanation

**Comment by becka11y** [comment]
I need more information to resolve this. I am guessing this should be a formal citation rather than just a link?
I did add a paragraph: 
<p>Where the same value exists in the WCAG 2.1 <a href="https://www.w3.org/TR/WCAG21/#input-purposes">Input Purposes for User Interface Components</a> values, this specification uses that definition.</p> 
before the table of values. I suspect that may not be enough. 
If we need a citation, I'll need help on the formatting. I could not find where to add WCAG 2.1 to our Reference list.

**Comment by lseeman** [comment]
roy is working on it....

**Comment by ruoxiran** [comment]
Added in https://github.com/w3c/personalization-semantics/commit/26ca604dff2bedbba88715214035014e34947fb9
